src/main/java/Main.java

```diff
@@ -0,0 +1,30 @@
+import java.io.IOException;
```

Comment:
Can try to split imports based on which package they are from for better readability